blog.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Blogs</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+    <div class="flex-container">
+        <div class="header"
+        style="background-image: url('images/header-image.jpg')">
+          <h1>ADOPTADOG</h1>
+          <h2>Seattle's premiere dog adoption agency</h2>
+        </div>
+
+        <div class="navbar">
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="dogs.html">Dogs</a></li>
+              <li><a href="blog.html">Blog</a></li>
+              <li><a href="checkout.html">Checkout</a></li>
+            </ul>
+        </div>
+
+        <div class="blogpost">
+          <h1>Adoptadog Blog</h1>
+          <img src="images/home-intro.jpg" alt="a man hugging his dog">
+          <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus
+    am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate</p>
+
+          <div class="tdog">
```

Comment:
you will want to use the same class for all blog posts, that way you don't need to create a new CSS class for each since they are all styled the same